### ISSUE TITLE
Automated Push Messages 

### ISSUE THREAD

**Issue by Nur-alfayez** [EmanHylooz/Abjjad-Product-Marketing#72]
Hey girls, 

Here are suggested automated push messages, mostly engagement related. I'll add more next week related to subscription and free trial. Your thoughts please, @EmanHylooz @Farahalfayez 

**- Signed in, and didn't download a book within 1 days >** (not on free trial, not subscribed, not canceled/expired) 

**حمّل كتابك الأول!** 
اختر من بين ٧٠٠ كتاب مجاني.. حمّل كتابك الأول الآن

_(deep link: take them to free books page)_

**- Didn't open app in 7 days** 

**آلاف الكتب في انتظارك** 
تشغلنا الحياة عن ما نحب، خصص بعض من وقتك للقراءة :) 

_(deep link: take them to books page)_
*Only show once per user

**- Clicked on notify me when available ** 

** شخص قام بالتصويت على هذا الكتاب x** 
شكراً لك، نعطي الأولوية لإضافة الكتب حسب تصويت قرّائنا. سنقوم بإعلامك حين توفر الكتاب. :) 

_(link: re-launch app)_
*Only show once per user 

**- A new book was added, (user added to I'll read it, or clicked on notify me once available)** 

** قمنا بإضافة الكتاب الذي تريده!** 
بناءً على طلبكم، قمنا بإضافة كتاب (    )، حمّله الآن

_(deep link: take them to the book page)_

**- A new book was added, if user doesn't have it on (I'll read it, or notify me)** 

** أضفنا كتاب (اسم الكتاب)** 
أضفنا كتاب (اسم الكتاب)، قم بتحميله الآن

**- User finished reading book** 

** هل أعجبك (اسم الكتاب؟)** 
قم بتقييه وإعطاء رأيك فيه عن طريق كتابة مراجعتك لتساعد الأبجديين في اخيار أفضل الكتب

_(deep link: take them to أضف مراجعة for the same book)_

**- Someone wrote a review on a book, send their friends** 

** (اسم الشخص) كتب مراجعة عن (اسم الكتاب)** 
هل قرأت (اسم الكتاب)؟ اقرأ ما كتب (اسم الشخص) عنه، هل توافقه؟

_(deep link: take them to the review)_

**- User read less than 50% of a book, didn't come back for a week** 

** لم يعجبك (اسم الكتاب)؟* 
جرّب (اسم كتاب من نفس التصنيف) - random from same filters (same category/free/freemium/top read) 

_(deep link: take them to suggested book page)_

**- User read more than 50% of a book, didn't come back for a week** 

** أكمل قراءة (اسم الكتاب)؟* 
إنه لشعور محبط أن تتوقف في منتصف الطريق! لقد أوشكت على إنهاء قراءة (اسم الكتاب).. أكمل قراءته :)

_(deep link: take them to suggested book page)_

**- Active (finished reading at least 1 free book, logs in on weekly basis) but never tried free trial** 

**هل تودّ القراءة بلا حدود* 
ليس لديك ما تخسره! ابدأ القراءة بلا حدود لشهر مجاناً وتوقف في أي وقت.. جرّبه الآن

_(deep link: take them to subscribe page)_

**Comment by EmanHylooz** [comment]
Let me take a deeper look and get back to you on Sunday.

Best,

Eman Hylooz

Founder & CEO | Abjjad

Jordan: +962.79.5402965 | Lebanon: +961.81.667918 | *www.abjjad.com*
<http://www.abjjad.com/>

2017-09-07 17:13 GMT+03:00 Nur-alfayez <notifications@github.com>:

**Comment by EmanHylooz** [comment]
Hello @Nur-alfayez , they look good, maybe few changes are needed based on targeting and some technical stuff I have in mind. Better brainstorm about them once you are done?

**Comment by Nur-alfayez** [comment]
Sure, will revise and keep you updated so we brainstorm.

**Comment by EmanHylooz** [comment]
(y)

Best,

Eman Hylooz

Founder & CEO | Abjjad

Jordan: +962.79.5402965 | Lebanon: +961.81.667918 | *www.abjjad.com*
<http://www.abjjad.com/>

**Comment by Nur-alfayez** [comment]
- User starts free trial 

**أهلا بك! لقد بدأت شهرك المجاني في "أبجد بلا حدود"**
اشتراك "أبجد بلا حدود" الشهري يوفر لك آلاف الكتب بالتعاون مع دور النشر: حمّل الكتب بلا حدود - اقرأ دون الحاجة للانترنت 

(deep link: take them to books page)


- User cancels subscription 

**نتمنى أن تبقى معنا** 
لقد قمت بإلغاء اشتراكك في "أبجد بلا حدود"، قم بتحريك هاتفك للتواصل معنا مباشرة بخصوص اشتراكك 

- User cancels subscription 

**نتمنى أن تبقى معنا** 
لقد قمت بإلغاء اشتراكك في "أبجد بلا حدود"، قم بتحريك هاتفك للتواصل معنا مباشرة بخصوص اشتراكك 

(link: re-launch app)

- User cancels subscription, does not open app in 7 days 

ما زال لديك ٧٠٠ كتاب مجاني!
لا تنسى بأن لديك ٧٠٠ كتاب متاح للقراءة مجاناً، كما يمكنك إعادة تفعيل اشتراكك في"أبجد بلا حدود" في أي وقت

(link: free books)

**Comment by Nur-alfayez** [comment]
@EmanHylooz I added some above.

**Comment by EmanHylooz** [comment]
Great, shall we consider this the final scope to review it and also prepare
the matching automated newsletters for it (after reviewing the
notifications)?

Best,

Eman Hylooz

Founder & CEO | Abjjad

Jordan: +962.79.5402965 | Lebanon: +961.81.667918 | *www.abjjad.com*
<http://www.abjjad.com/>

**Comment by Nur-alfayez** [comment]
Yeah, if we'll do a matching automated newsletter scope I'd review them both together so they go along instead of duplicating content. At least text wise.

**Comment by EmanHylooz** [comment]
We need to review the logic for notifications.. text can be left the last
step. But to review logic need you to further explain a couple of things so
I will be sure its for sure how i understood it.
Shall we set an hour to go through it?

**Comment by Nur-alfayez** [comment]
Sure, lets do so tomorrow after Zahra's call.

**Comment by EmanHylooz** [comment]
Sounds great.

On Sep 11, 2017 15:26, "Nur-alfayez" <notifications@github.com> wrote:

Sure, lets do so tomorrow after Zahra's call.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/EmanHylooz/Abjjad-Product-Marketing/issues/72#issuecomment-328513141>,
or mute the thread
